src/com/itacademy/eshop/product/TV.java

```diff
@@ -0,0 +1,23 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public class TV extends Electronics{
+
+    public boolean available;
+    public TV(String name, double price, Category category, String voltage, String brand, boolean available) {
+        super(name, price, category, voltage, brand);
+        this.available = available;
+    }
+
+    public boolean isAvailable(){ return false; }
```

Comment:
should return field is available value, and not just false, with this implementation all tv's are unavailable